ui/app/components/app/transaction-breakdown/transaction-breakdown.component.js

```diff
@@ -29,13 +30,23 @@ export default class TransactionBreakdown extends PureComponent {
 
   render () {
     const { t } = this.context
-    const { gas, gasPrice, value, className, nativeCurrency, showFiat, totalInHex, gasUsed } = this.props
-
+    const { gas, gasPrice, value, className, nonce, nativeCurrency, showFiat, totalInHex, gasUsed } = this.props
+    console.log(nonce)
     return (
       <div className={classnames('transaction-breakdown', className)}>
         <div className="transaction-breakdown__title">
           { t('transaction') }
         </div>
+        <TransactionBreakdownRow title="Nonce">
+          {typeof nonce !== 'undefined'
+            ? (
+              <HexToDecimal
+                className="transaction-breakdown__value"
+                value={nonce}
+              />
+            ) : '?'
```

Comment:
I would recommend using "Unknown" here, since we already have it as a localized message

```suggestion
            ) : t('unknown')
```

Reply:
Or just leave it blank, perhaps? :thinking: 

```suggestion
            ) : null
```